SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.h

```diff
@@ -324,6 +324,8 @@ class SiPixelDigitizerAlgorithm  {
 
     const double electronsPerVCAL;          // for electrons - VCAL conversion
     const double electronsPerVCAL_Offset;   // in misscalibrate()
+    const double electronsPerVCAL_L1;       // same for Layer 1
```

Comment:
if float are used in the algo, please use float here as well (conversion IS expensive)